sdk/helper/template/template_test.go

```diff
@@ -207,3 +210,56 @@ func TestBadConstructorArguments(t *testing.T) {
 		require.Equal(t, "", str)
 	})
 }
+
+func TestTemplateInputLength(t *testing.T) {
```

Comment:
Do we encounter the same error that was reported without the changes in `template.go`? Wondering since that was discovered via fuzzing, and our test here is written slightly differently.